src/eventHandlers/SplitsSetEvent/processLinkedIdentitySplits.ts

```diff
@@ -39,6 +46,46 @@ export async function setLinkedIdentityFlag(
     linkedIdentity.ownerAccountId,
   );
 
+  assertIsRepoDriverId(accountId);
+
+  const existingSplits = await SplitsReceiverModel.findAll({
+    where: { senderAccountId: accountId },
+    transaction,
+  });
+
+  // ORCID accounts should have exactly one splits receiver (100% to owner).
+  if (existingSplits.length > 1) {
+    const errorMsg = `Found ${existingSplits.length} splits receivers for ORCID account ${accountId}, expected 1`;
+    scopedLogger.bufferMessage(errorMsg);
+    throw new Error(errorMsg);
+  }
+
+  if (existingSplits.length > 0) {
```

Comment:
what if it's 0? then we don't destroy the possibly existing old receiver but also create a new one?